ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -258,6 +259,50 @@ def test_paddle_ifftshift(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.fft.ihfft2",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=["float64"],
```

Comment:
Can we sample all supported dtypes as suggested by the function wrapper on the implementation side

Reply:
When I used all the supported dtypes in the test function, I was getting an Assertion error. Something like "returned dtype is complex64 but ground truth returned dtype is complex128 ". Then when I cast the return type to complex128, the assertion error switches into "returned dtype is complex128 but ground truth returned dtype is complex64"(just interchanging the places of complex64 and complex128). 

Is it better if I change the supported dtypes on the implementation side to only "float64", or include all the supported dtypes in the test function wrapper(@handle_frontend_test) even if the assertion error will show up?

Reply:
This should be handled by promoting due to correct precision `complex` dtype. You can maintain the conversion of result to be `complex64` if input is `int32`, `float32` else if the input is `float64` and `int64` we can cast the result to `complex128`

Reply:
Hey @hirwa-nshuti, I have included all the supported types in the test function and handled the conversion of the correct precision complex dtype for each corresponding input dtype as you suggested. All 5 tests are passing as well.